runtime/gc_modron_startup/mminit.cpp

```diff
@@ -2867,7 +2869,9 @@ configurateGCWithPolicyAndOptions(OMR_VM* omrVM)
 		extensions->gcModeString = "-Xgcpolicy:gencon";
 		omrVM->gcPolicy = J9_GC_POLICY_GENCON;
 		/* scavenge, concurrentMark, noConcurrentSweep, loa */
-		setDefaultConfigOptions(extensions, true, true, false, true);
+		setDefaultConfigOptions(extensions, true,
+			(1 != j9sysinfo_get_number_CPUs_by_type(J9PORT_CPU_TARGET)),
```

Comment:
Yes, I think that's right way to do it.
Just break it into 2 lines for easier reading. Introduce a temp bool concurrentMark.

Reply:
change it as suggested.